cfgov/unprocessed/js/molecules/Multiselect.js

```diff
@@ -21,6 +21,14 @@ var strings = require( '../modules/util/strings' );
  */
 function Multiselect( element ) { // eslint-disable-line max-statements, inline-comments, max-len
 
+  var BASE_CLASS = 'cf-multi-select';
+
+  // TODO: Add 'cf-multi-select' class to <select> menu markup
```

Comment:
Possibly, though I'd lean toward no. Leave the js-hook for things that don't do any of their own styling. This is kind of a weird case because it's saying "style this thing when it gets swapped out with a div," so the select isn't receiving styles but it's swapped with something that does. It seems unclear to shuffle from a js-hook attribute to a class during the swap. And attaching styles to the js-hook is counter to its intended use. 
 What would be the downside of scoping the styles to `div.cf-multi-select`? Also, potentially there could be fallback styles that would be useful at `select.cf-multi-select`